app/helpers/result_helper.rb

```diff
@@ -0,0 +1,17 @@
+module ResultHelper
+  def title_string(object, kind)
+    invalid_date = kind == 'conviction' ? 'never' : 'caution_result'
+    return t("#{kind}_title.#{invalid_date}", scope: "results/#{kind}") unless object.instance_of?(Date)
+
+    title_type = object < Date.today ? 'past' : 'present'
+    t("#{kind}_title.#{title_type}", scope: "results/#{kind}", date: I18n.l(object))
```

Comment:
Following the above comment, here we don't really need to prefix the locale `"#{kind}_title` as we are already scoping them. So just `title.no_date` would be enough.

If we were to be super sure in case in the future we add an attribute to the DB named `title` or similar, we can extend the scope to read: `scope: "results/#{kind}"/confirmation_panel`